### ISSUE TITLE
[Bug] The name 'WeekLayout' does not exist in the current context

### ISSUE THREAD

**Issue by kerberosargos** [lilcodelab/Xamarin.Plugin.Calendar#113]
Hi folks, I have installed v1.4.5304 from nuget but It does not contains **Enums**. 

```
using Xamarin.Plugin.Calendar.Enums;

private WeekLayout _calendarLayout = WeekLayout.Month;

```
```
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0103	The name 'WeekLayout' does not exist in the current context	
```

**Comment by kvpt** [comment]
From the git history, the WeekLayout has been introduced by the PR #106 which has been merged the 11 october and the last nuget package has been published the 8 july.
So you will need to wait for the next release or build your own package from the source.

**Comment by aodegov** [comment]
Issue still exists:

![image](https://user-images.githubusercontent.com/24685526/154810310-fa48e0b0-db7a-476f-a2eb-7c8d7cacbb8f.png)

**Comment by wadebaird** [comment]
@aodegov : Yes, it will still exist. Please see @kvpt 's response above.

**Comment by mcblacksea** [comment]
@wadebaird, @kerberosargos , @kvpt , as far as I can see this issue has been fixed with latest NuGet package updates.
At least I don't have this anymore.
Could you please check on your side. 
And then, if  everything is alright this issue can be closed.

**Comment by kerberosargos** [comment]
Yes. This issue has gone. You can close. Thank you for your efforts 👏

28 May 2022 Cmt, saat 01:33 tarihinde aodegov ***@***.***>
şunu yazdı:

**Comment by wadebaird** [comment]
Yes, I agree, this can be closed.
